content/en/docs/external-add-ons/_index.md

```diff
@@ -1,5 +1,9 @@
 +++
 title = "External Add-Ons"
-description = "Additional tools that may be integrated with a Kubeflow deployment or distribution."
+description = "Externally developed projects that integrate with Kubeflow"
 weight = 30
-+++ 
\ No newline at end of file
++++
+
+{{% alert title="Add-On Ownership" color="warning" %}}
```

Comment:
Should we use other colours rather than red here (e.g. Yellow)?
We use red colour to show beta or legacy supported distributions which might confuse users.
For example, Kubernetes uses yellow colour for other tools: https://kubernetes.io/docs/reference/tools/#minikube.
cc @akgraner @jbottum